app/views/spree/payment_mailer/authorize_payment.text.erb

```diff
@@ -0,0 +1,2 @@
+<%= t('.instructions', distributor: @payment.order.distributor.name, amount: @payment.display_amount) %>
+<%= main_app.authorize_payment_url(@payment) %>
```

Comment:
Can we make this HAML instead? We don't really use ERB.

Reply:
c71e5037129c43f2431c4ab4037af7cea20f6e88 👍